web/templates/room/_room.html.eex

```diff
@@ -1,3 +1,23 @@
+<%= cond do %>
```

Comment:
if there is only two conditions I prefer to use ```if/else```. It's what [credo](https://github.com/rrrene/credo) also recommend I think